### ISSUE TITLE
dependency listings in docs

### ISSUE THREAD

**Issue by sosey** [spacetelescope/jwst#2536]
The `assign_wcs` docs have a dependency listing: https://github.com/spacetelescope/jwst/blob/master/docs/jwst/assign_wcs/main.rst

We should consider removing this and creating a dependency entry for the whole package at a higher level in the docs that's pulled in automatically if possible. (unless I've already missed that somewhere?)

**Comment by nden** [comment]
I agree this should be removed. We need a dependency listing for the entire jwst package.

**Comment by stscicrawford** [comment]
Related to #1508
